open_spiel/games/universal_poker.cc

```diff
@@ -709,19 +735,6 @@ int UniversalPokerGame::MaxGameLength() const {
  */
 std::string UniversalPokerGame::parseParameters(const GameParameters &map) {
   if (map.find("gamedef") != map.end()) {
-    // We check for sanity that all parameters are empty
-    if (map.size() != 1) {
```

Comment:
Had to remove this to allow passing `cardAbstraction` along with `gamedef`. 

The other option is to pass everything in `gamedef` as a single game parameter which is cumbersome. I think this is how `bettingAbstraction` is supposed to be used as well?

Reply:
I think this is fine as-is.

Reply:
i.e. removing it is fine.

Reply:
Actually, no, can you modify to check that the only other field is `cardAbstraction`?